make_printable.sh

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env bash
 
+LANGUAGE=$1
+en=1
+pl=2
+declare -p PATHS
+declare -a PATHS=( [1]="sections" [2]="sections/translated/pl" )
```

Comment:
Why does it hardcode pl here?

Reply:
Let me add Spanish to better demonstrate this.

Reply:
See the updated version. I wanted to choose the path with the input argument. I used bash associative array for this. I'm open for ideas.

Reply:
See the suggestion. No need to list the languages here, and no need for this indirection through numbers